vm/devices/storage/nvme_test/src/workers/admin.rs

```diff
@@ -505,6 +505,12 @@ impl AdminHandler {
                                 &command, &message
                             );
                         }
+                        QueueFaultBehavior::CustomPayload(_) => {
+                            panic!(
+                                "bad fault configuration: custom payloads are not applicable to admin submission commands. command: {:?}",
+                                &command
+                            );
+                        }
```

Comment:
Is this a temporary constraint? I would expect us to want this eventually (for example, to change some downstream behavior by changing the supplied parameters). Perhaps I am mistaken and there aren't any admin commands we care about that have external data.

Reply:
Also, as a future-looking comment: it would be better to catch this in test construction than what is essentially runtime here.